tests/integration/telemetry/tracing/otelcollector/tracing_test.go

```diff
@@ -53,14 +53,15 @@ func TestProxyTracingOpenCensusMeshConfig(t *testing.T) {
 						if err != nil {
 							return fmt.Errorf("cannot send traffic from cluster %s: %v", cluster.Name(), err)
 						}
-
-						traces, err := tracing.GetZipkinInstance().QueryTraces(300,
-							fmt.Sprintf("server.%s.svc.cluster.local:80/*", appNsInst.Name()), "")
-						if err != nil {
-							return fmt.Errorf("cannot get traces from zipkin: %v", err)
-						}
-						if !tracing.VerifyEchoTraces(ctx, appNsInst.Name(), cluster.Name(), traces) {
-							return errors.New("cannot find expected traces")
+						for _, zipkinInst := range tracing.GetZipkinInstances() {
```

Comment:
Not sure this really makes sense, why do we need to query all of them? They are the same pod, seems redundant?

Reply:
Testing Istio functionality with dual stack is important, but testing 3rd party seems redundant?

Reply:
I agree no need to test that. I have changed back the telemetry tests to no longer test query of traces for both addresses.
The changes to retrieve all the Zipkin addresses are still in place to remain consistent with the other getters